src/store.ts

```diff
@@ -109,7 +118,10 @@ export function addCVR(batchId: number, filename: string, cvr: CastVoteRecord) {
     'insert into CVRs (batch_id, filename, cvr_json) values (?, ?, ?)',
     batchId,
     filename,
-    JSON.stringify(cvr)
+    JSON.stringify(cvr),
+    () => {
+      // we tolerate an error here because occasionally there are dupe inserts
```

Comment:
nit: Do it or TODO it!

Reply:
we are doing it, that's what the callback does: it ignores the error. I'll clarify the comment.